src/BasePlatform.js

```diff
@@ -65,4 +65,12 @@ export default class BasePlatform {
 
     displayNotification(title: string, msg: string, avatarUrl: string) {
     }
+
+    /**
+     * Returns a promise that resolves to a string representing
+     * the current version of the application.
+     */
+    getAppVersion() {
+        return q();
```

Comment:
how about making this throw an error, so that people don't forget to implement it?